open_ce/get_graph.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+"""
+# *****************************************************************
+# (C) Copyright IBM Corp. 2021. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *****************************************************************
+"""
+
+import os
+
+from open_ce.inputs import Argument
+from open_ce import graph
+from open_ce import build_env
+from open_ce import utils
+
+COMMAND = "graph"
+
+DESCRIPTION = 'Plot a dependency graph.'
+
+ARGUMENTS = build_env.ARGUMENTS + \
```

Comment:
Should this be `inputs.ENV_BUILD_ARGS` instead of `build_env.ARGUMENTS`?

Reply:
I had that at first, but I want to make sure that if we update the build_env args they get updated here too. I want it to use the exact same args as build_env.

Reply:
OK, that is fine. There will just be some arguments that don't make as much sense such as `--skip_build_packages`, but that should be fine.

Reply:
After going back and looking at it, you're right, I had it backwards.